app/_data/docs_nav_gateway_3.0.x.yml

```
@@ -74,117 +68,131 @@ items:
           - text: DB-less Deployment
-            url: /kong-production/deployment-topologies/db-less-and-declarative-config
+            url: /production-deployment/deployment-topologies/db-less-and-declarative-config
           - text: Traditional
-            url: /kong-production/deployment-topologies/traditional
+            url: /production-deployment/deployment-topologies/traditional
       - text: Installation Options
         items:
           - text: Kubernetes
-            url: /kong-production/install-options/kubernetes
-          - text: Helm
-            url: /kong-production/install-options/helm
-          - text: OpenShift with Helm
-            url: /kong-production/install-options/openshift
+            items:
+              - text: Install Kong Gateway with Kubernetes
+                url: /install/kubernetes/kubernetes
+              - text: Helm
+                url: /install/kubernetes/helm
+              - text: OpenShift with Helm
+                url: /install/kubernetes/openshift
+              - text: Kubernetes Deployment Options
+                url: /install/kubernetes/deployment-options
           - text: Docker
             items:
               - text: Install Kong with Docker
-                url: /kong-production/install-options/docker
+                url: /install/docker
               - text: Build your own Docker images
-                url: /kong-production/install-options/docker/build-custom-images/
+                url: /install/docker/build-custom-images/
           - text: Linux
             items:
               - text: Supported Distributions
-                url: /kong-production/install-options/linux/os-support
+                url: /install/linux/os-support
               - text: Amazon Linux
-                url: /kong-production/install-options/linux/amazon-linux
+                url: /install/linux/amazon-linux
               - text: Debian
-                url: /kong-production/install-options/linux/debian
+                url: /install/linux/debian
               - text: Red Hat
-                url: /kong-production/install-options/linux/rhel
+                url: /install/linux/rhel
               - text: Ubuntu
-                url: /kong-production/install-options/linux/ubuntu
+                url: /install/linux/ubuntu
           - text: macOS
-            url: /kong-production/install-options/macos
+            url: /install/macos
       - text: Running Kong
         items:
-          - text: Start Kong Gateway Securely
-            url: /kong-production/running-kong/start-kong-securely
           - text: Running Kong as a non-root user
-            url: /kong-production/running-kong/kong-user
+            url: /production-deployment/running-kong/kong-user
+          - text: Securing the Admin API
+            url: /production-deployment/running-kong/secure-admin-api
           - text: Configure Kong Gateway and SystemD
```

```suggestion
          - text: Configure Kong Gateway and systemd
```
The name of systemd is all lowercase.